submissions/chelzakaria/main.py

```diff
@@ -0,0 +1,32 @@
+import polars as pl
```

Comment:
Is it an external lib?

Reply:
Yes ! Isn't allowed ?

Reply:
No. Only standard lib can be used

Reply:
Alright then. But it should be mentioned in the rules

Reply:
Rules updated to reflect this